tests/char_biometrics_test.cpp

```diff
@@ -228,94 +264,96 @@ TEST_CASE( "body mass index determines maximum healthiness", "[biometrics][bmi][
+        test_relative_heights( dummies_default_height, dummies_min_height,
+                               creature_size::medium, creature_size::huge );
+    }
 
-    GIVEN( "character height strarted at 160cm" ) {
-        CHECK( height_with_base_and_size( dummy, 160, "MEDIUM" ) == 160 );
-        // Always 30 cm shorter than max for each size class
-        CHECK( height_with_base_and_size( dummy, 160, "SMALL" ) == 110 );
-        CHECK( height_with_base_and_size( dummy, 160, "LARGE" ) == 210 );
-        CHECK( height_with_base_and_size( dummy, 160, "HUGE" ) == 260 );
+    GIVEN( "default height character vs max height character" ) {
+        test_relative_heights( dummies_default_height, dummies_max_height,
+                               creature_size::medium, creature_size::tiny );
+        test_relative_heights( dummies_default_height, dummies_max_height,
+                               creature_size::medium, creature_size::small );
+        test_relative_heights( dummies_default_height, dummies_max_height,
+                               creature_size::medium, creature_size::large );
+        test_relative_heights( dummies_default_height, dummies_max_height,
+                               creature_size::medium, creature_size::huge );
     }
 
-    SECTION( "character height started at 190cm" ) {
-        CHECK( height_with_base_and_size( dummy, 190, "MEDIUM" ) == 190 );
-        // Always at maximum height for each size class
-        CHECK( height_with_base_and_size( dummy, 190, "SMALL" ) == 140 );
-        CHECK( height_with_base_and_size( dummy, 190, "LARGE" ) == 240 );
-        CHECK( height_with_base_and_size( dummy, 190, "HUGE" ) == 290 );
+    GIVEN( "min height character vs max height character" ) {
+        test_relative_heights( dummies_min_height, dummies_max_height,
+                               creature_size::medium, creature_size::tiny );
+        test_relative_heights( dummies_min_height, dummies_max_height,
+                               creature_size::medium, creature_size::small );
+        test_relative_heights( dummies_min_height, dummies_max_height,
+                               creature_size::medium, creature_size::large );
+        test_relative_heights( dummies_min_height, dummies_max_height,
+                               creature_size::medium, creature_size::huge );
     }
 }
 
-TEST_CASE( "size and height determine body weight", "[biometrics][bodyweight]" )
+TEST_CASE( "default character (175 cm) bodyweights at various BMIs", "[biometrics][bodyweight]" )
 {
     avatar dummy;
     clear_character( dummy );
 
-    // Body weight is calculated as ( BMI * (height/100)^2 ). At any given height, body weight
-    // varies based on body mass index (which in turn depends on the amount of stored calories).
-
-    // Check bodyweights at each size for two different heights:
-    // 175cm (original default) and 190cm (maximum for MEDIUM sized human)
-
     GIVEN( "character height started at 175cm" ) {
```

Comment:
Why remove the first comment above re BMI?

Reply:
Whoops.